src/core/Normalization.helper.js

```diff
@@ -29,7 +29,26 @@ export default class NormalizationHelper {
     }
     normalized = Number(normalized.toFixed(0));
     return normalized;
+  }
 
+  /**
+   * @param {Number|null} value
+   * @returns {Number|null}
+   */
+  static normalizeNonZeroPositiveInteger (value) {
+    if (typeof value === 'string' || value instanceof String) {
+      /** If is string */
+      if (/^((\d*\.\d+)|(\d+))$/g.test(value)) {
+        if (value !== '0') {
```

Comment:
@robertdumitrescu INFO will this catch '0.0'?

Reply:
Good call. Will add that.

Reply:
OK - done